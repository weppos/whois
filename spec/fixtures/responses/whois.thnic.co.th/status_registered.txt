
Whois Server Version 2.1.0

Domain: GOOGLE.CO.TH
Registrar: T.H.NIC Co., Ltd.
Name Server: NS1.GOOGLE.COM
Name Server: NS2.GOOGLE.COM
Name Server: NS3.GOOGLE.COM
Name Server: NS4.GOOGLE.COM
Status: ACTIVE
Updated date: 7 Sep 2012
Created date: 8 Oct 2004
Renew date: 8 Oct 2012
Exp date: 7 Oct 2013
Domain Holder: Google Inc.
2400 Bayshore Parkway, Mountain Veiw, CA
94043
US

Tech Contact: 13244
MarkMonitor Inc.
391 N Ancestor Place, Boise, ID
83704
US