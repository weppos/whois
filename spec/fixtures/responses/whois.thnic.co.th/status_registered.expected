#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2004-10-08")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-09-07")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-10-07")


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.google.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.google.com"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.google.com"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.google.com"
